src/frontend/src/components/VerticalDetailDisplay.tsx

```diff
@@ -16,13 +16,34 @@ interface VerticalDetailDisplayProps {
 const VerticalDetailDisplay: React.FC<VerticalDetailDisplayProps> = ({ label, content, boxStyle }) => {
   const theme = useTheme();
   const backgroundColor = theme.palette.mode === 'dark' ? theme.palette.grey[800] : theme.palette.grey[200];
+
+  // Box Styles
+  const customSx = {
```

Comment:
This component has a prop called `boxStyle`. This prop should still be used; switch the argument used for this prop when it is implemented instead of writing it out here